chia/cmds/show.py

```diff
@@ -57,236 +89,305 @@ async def print_connections(client, time, NodeType, trusted_peers: Dict):
         print(con_str)
 
 
+async def execute_with_node(rpc_port: Optional[int], function: Callable, *args):
+    import traceback
+
+    import aiohttp
+
+    from chia.util.config import load_config
+    from chia.util.default_root import DEFAULT_ROOT_PATH
+    from chia.util.ints import uint16
+    from chia.rpc.full_node_rpc_client import FullNodeRpcClient
+
+    try:
+        config = load_config(DEFAULT_ROOT_PATH, "config.yaml")
```

Comment:
we should use the config loading methods here now

Reply:
Im not sure what you mean by that.